lighthouse-cli/test/smokehouse/smokehouse.js

```diff
@@ -34,8 +34,11 @@ const PROTOCOL_TIMEOUT_EXIT_CODE = 67;
 const PAGE_HUNG_EXIT_CODE = 68;
 const INSECURE_DOCUMENT_REQUEST_EXIT_CODE = 69;
 const RETRIES = 3;
-const NUMERICAL_EXPECTATION_REGEXP = /^(<=?|>=?)((\d|\.)+)$/;
 const VERBOSE = Boolean(process.env.LH_SMOKE_VERBOSE);
+const NUMBER_REGEXP = /(?:\d|\.)+/.source;
+const OPS_REGEXP = /<=?|>=?|\+\/-/.source;
+const NUMERICAL_EXPECTATION_REGEXP =
+  new RegExp(`^(${NUMBER_REGEXP})?\\s?(${OPS_REGEXP})\\s?(${NUMBER_REGEXP})$`);
```

Comment:
is this clearer than `^((?:\d|\.)+)?\s?(<=?|>=?|\+\/-)\s?((?:\d|\.)+)$`

Reply:
yes, but I'm not sure I'd call either of them super clear ;)

maybe we can at least get a comment here explaining what's going on in english?

`An optional number, single optional whitespace character, an operator, single optional whitespace character, a number.`